plugins/NBBC/class.nbbc.plugin.php

```diff
@@ -219,7 +219,8 @@ function DoURL($bbcode, $action, $name, $default, $params, $content) {
    public function Format($Result) {
       $Result = str_replace(array('[CODE]', '[/CODE]'), array('[code]', '[/code]'), $Result);
       $Result = $this->NBBC()->Parse($Result);
-      return $Result;
+      $Mixed = html_entity_decode($Result);
+      return Gdn_Format::html($Mixed);
```

Comment:
The call to **Gdn_Format::html()** will strip some css classes out that are appropriate for the BBCode parser.